### ISSUE TITLE
`search`: duplicate package name with `NO_COLOR`

### ISSUE THREAD

**Issue by cyrinux** [AladW/aurutils#995]
Hi,

with aurutils-git, `NO_COLOR=true` is broken.

the package name is duplicate.

```
$ NO_COLOR=true aur search -n -k NumVotes arch-secure-boot
aur/arch-secure-bootaur/arch-secure-boot 1.3.2-1 (+2 0.00%)
    UEFI Secure Boot for Arch Linux + btrfs snapshot recovery

$ aur search -n -k NumVotes arch-secure-boot
aur/arch-secure-boot 1.3.2-1 (+2 0.00%)
    UEFI Secure Boot for Arch Linux + btrfs snapshot recovery
```

Regards

**Comment by Earnestly** [comment]
Seems like missing newline in no_color codepath

**Comment by cyrinux** [comment]
Hmm, i'm using kitty, with color enable, i can see the get the link when mouse over `aur/arch-secure-boot` and with NO_COLOR=true i can get the link on the second duplicate `aur/arch-secure-boot`.

**Comment by Earnestly** [comment]
Yes, you should it's always used.  Not quite sure why it's only doubling the name, as it's not missing elements that would otherwise interfere with printf(1)'s implicit looping feature.

**Comment by AladW** [comment]
The issue appears to be that the package name is printed a second time when `ALL_OFF` is undefined i.e. if `colorize()` was not run. The function should return in that case:
```bash
if [[ ! -v ALL_OFF ]]; then
    printf "%s" "$mesg"
    return  # <---
fi
```
That said, I'm not sure `NO_COLOR` should disable the link since it's only about "color" escapes (which are e.g. annoying when debugging with `set -x`).

**Comment by AladW** [comment]
Fixed with #996
